drivers/virtualbox/vbm.go

```diff
@@ -15,8 +15,6 @@ import (
 )
 
 var (
-	reVMNameUUID      = regexp.MustCompile(`"(.+)" {([0-9a-f-]+)}`)
```

Comment:
Fair, but we should run by github.com/boot2docker/boot2docker-cli to see if it was something relevant which was accidentally left out in the port to Machine.